.github/workflows/deployDemo.yml

```diff
@@ -27,18 +27,30 @@ jobs:
         uses: docker/setup-qemu-action@v1
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
+      - uses: azure/login@v1
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
       - name: Login to ACR
         run: docker login ${{ secrets.ACR_REPO_URL }} -u ${{ secrets.ACR_ADMIN_USERNAME }} -p ${{ secrets.ACR_ADMIN_PASWORD }}
       - name: Build and push Docker images
         run: |
           ./build_and_push.sh
       - uses: hashicorp/setup-terraform@v1
+        with:
+          terraform_version: 0.14.10
       - name: Terraform Init
         working-directory: ${{ github.workspace }}/ops/${{ env.DEPLOY_ENV }}
         run: terraform init
       - name: Azure deploy Api
         working-directory: ${{ github.workspace }}/ops/${{ env.DEPLOY_ENV }}
-        run: terraform apply -auto-approve -var="acr_image_tag=${GITHUB_SHA::7}" -target="module.simple_report_api.azurerm_app_service.service"
+        run: terraform apply -auto-approve -var="acr_image_tag=${GITHUB_SHA::7}" -target="module.simple_report_api"
```

Comment:
Is this intended to deploy out the whole module now instead of the scoped module?

Reply:
Yes, we want to deploy everything owned by the app-service module—usually that will just be the app-service-slot, but if we change something else about the way the API is deployed, we shouldn't fail to deploy that as well. (And this way if we are bringing this up in a cold-start environment, it should still work.)